### DISCUSSION TITLE
Errors with three subdomains mesh with interface transfer

### DISCUSSION THREAD

**Discussion by salaudeen-ya** [idaholab/moose#22323] [Q&A Meshing] [answered]
Hi all,

I am trying to have three subdomains for a simple BVP. My two-domain works just fine, but I have errors with three subdomains.

My input file and error message are attached.
Thanks

Kindly help.
![input](https://user-images.githubusercontent.com/38023107/194422285-1c6afecc-5812-4f8e-8dd3-f93fd295b7f0.png)
![error](https://user-images.githubusercontent.com/38023107/194422286-76b46cb6-ab2c-48e9-85d6-e0818956d37e.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

You made a tree of mesh generators that has multiple outputs. 
Try chaining all the mesh generators, making each MG the `input =` of the one below it

Guillaume

**Reply by salaudeen-ya** [reply]
Wow!
That worked..

Thanks.
